### PR TITLE
[Merged by Bors] - fix: update to Fluvio 15

### PR DESCRIPTION
**Pull request by EstebanBorai**
Updates Fluvio version to v14. Also updates new imports.

### CODE REVIEW

File: src/consumer/mod.rs [hunk 1]
```diff
@@ -237,7 +237,7 @@ impl fmt::Debug for RecordJS {
     }
 }
 
-use fluvio::dataplane::ErrorCode;
```
Comment (morenol) [inline src/consumer/mod.rs:240]: Is this change needed? I wonder if we can avoid installing fluvio_protocol and just use the fluvio crate that is supposed to be reexporting these strucs

Comment (EstebanBorai) [reply src/consumer/mod.rs:240]: Let me recheck! Probably I didn't noticed the reexport on this case.

Comment (EstebanBorai) [reply src/consumer/mod.rs:240]: Okey export is aliased to `dataplate`! I'm updating..

Comment (EstebanBorai) [reply src/consumer/mod.rs:240]: Addressed on: [8a84cb6](https://github.com/infinyon/fluvio-client-node/pull/269/commits/8a84cb631e85e96d48b32fbe8714dbd38007a7bb)

**Review by simlay — APPROVED** [review]
I think CI was failing in #275 and #274 because it was a fluvio client vs platform mismatch.
